### PR TITLE
Bump Storybook dependencies to v3

### PR DESCRIPTION
**Pull request by chris-verclytte**

### CODE REVIEW

**Comment by chris-verclytte** [comment]
Is think tests failed because problem of OAuth login but I am not able to launch the build again, so if you can make it for me, it should be ok !

**Comment by ndelangen** [comment]
I'm on it! Sorry I missed this PR, if this passes, will release it ASAP! ♥️

File: README.md [hunk 1]
```diff
@@ -21,15 +21,14 @@ Then create a file called `addons.js` in your storybook config.
...
 import 'react-storybook-addon-backgrounds/register';
 ```
 
 Then write your stories like this:
 
 ```js
 import React from 'react';
-import { storiesOf } from "@kadira/storybook";
+import { storiesOf } from "@storybook/react";
 import backgrounds from "react-storybook-addon-backgrounds";
```
Comment (ndelangen) [inline README.md]: I'd like this package to be renamed to `@storybook/addon-backgrounds` But I'll do it.

Comment (chris-verclytte) [inline README.md]: Ok great ! Thanks for the review.

Comment (ndelangen) [inline README.md]: Thank you for the PR mate, i just released, please let me know if it works for you?

Comment (chris-verclytte) [inline README.md]: It seems to work for me. However, we should update the README and storybook helper with the new package naming.

Comment (ndelangen) [inline README.md]: Done!

**Pull request merged**